content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -38,11 +39,12 @@ If a {{< glossary_tooltip term_id="node" >}} dies, the Pods scheduled to that no
 are [scheduled for deletion](#pod-garbage-collection) after a timeout period.
 
 Pods do not, by themselves, self-heal. If a Pod is scheduled to a
-{{< glossary_tooltip text="node" term_id="node" >}} that then fails, the Pod is deleted; likewise, a Pod won't
-survive an eviction due to a lack of resources or Node maintenance. Kubernetes uses a
-higher-level abstraction, called a
-{{< glossary_tooltip term_id="controller" text="controller" >}}, that handles the work of
-managing the relatively disposable Pod instances.
+{{< glossary_tooltip text="node" term_id="node" >}} that then fails, the Pod is
+deleted; likewise, a Pod won't survive an eviction due to a lack of resources or
+Node maintenance. What Pods can do is self-heal on a local node. In order to have
```

Comment:
> What Pods can do is self-heal on a local node.

This is not right. When a Pod fails, a controller might replace it. It's up to that controller and / or the control plane to decide what scheduling constraints to apply, and ultimately where to place that replacement pod.

For example, when a node fails, the replacements for the pods that were running there are likely to end up on a different, healthy node.